karriereat/release-bot
# Features and fixes - group issue

- CHANGELOG.md
- New option to skip sslVerification for Gitlab #1
- Use `:releasegopher:` emoji for slack message avatar
- Check for `%NO-RELEASE%` in release message to skip notification

- Show versions without release message #4
- Shorter title text
- Use namespace+repo instead of repo name in title #3
